bigip/resource_bigip_ltm_node.go

```diff
@@ -211,7 +210,6 @@ func resourceBigipLtmNodeUpdate(d *schema.ResourceData, meta interface{}) error
 
```

Comment:
Please remove `d.SetId("")` from line 202, Exists function should not modify state.

https://github.com/terraform-providers/terraform-provider-bigip/issues/7